test/scenarios/11. sign/eth sign.csv

```diff
@@ -0,0 +1,14 @@
+Steps,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown.,
+2,Proceed to Unlock the wallet.,password (8 characters min).,"The Ether balance is shown on the overview.
+The wallet address is shown on the overview.
+The selected network is Ethereum Mainnet.",
+3,Switch networks to a test network.,e.g. Goerli.,"The Goerli balance is shown on the overview.
+The wallet address is shown on the overview.
+The selected network is Goerli.",
+4,Toggle eth_sign requests advanced setting,,,
```

Comment:
Can we add the steps needed to do this? opening the accounts options menu -> selecting settings -> advanced -> enabling eth_sign.

Reply:
Added, thanks!